data/core/macros/deprecated-utils.cfg

```diff
@@ -164,7 +164,7 @@ _"No gold carried over to the next scenario."#enddef
             {TRAIT_UNDEAD}
             {TRAIT_LOYAL}
         [/modifications]
-        overlays="misc/loyal-icon.png"
+        {IS_LOYAL}
```

Comment:
Does IS_LOYAL do anything useful any more?

If the unit already has TRAIT_LOYAL then that will add the overlay, and if it doesn't then IS_LOYAL will be a bug-in-waiting with the overlay disappearing when the unit levels up.

Reply:
in the current state of the pr it still adds the ´overlays="misc/loyal-icon.png"´ line, this is wip though, so im not 100% sure how i want ti to be in the end,